test/metabase/api/automagic_dashboards_test.clj

```diff
@@ -49,24 +49,19 @@
   ([template args revoke-fn validation-fn]
    (mt/with-test-user :rasta
      (with-dashboard-cleanup
-       (mt/with-full-data-perms-for-all-users!
-         (let [api-endpoint (apply format (str "automagic-dashboards/" template) args)
-               resp         (mt/user-http-request :rasta :get 200 api-endpoint)
-               _            (dashcards-schema-check (:dashcards resp))
-               result       (validation-fn resp)]
-           (when (and result
-                      (try
-                        (testing "Endpoint should return 403 if user does not have permissions"
-                          (data-perms/set-database-permission! (perms-group/all-users) (mt/id) :perms/data-access :no-self-service)
-                          (perms/revoke-data-perms! (perms-group/all-users) (mt/id))
+       (let [api-endpoint (apply format (str "automagic-dashboards/" template) args)
```

Comment:
Seems like `mt/with-full-data-perms-for-all-users!` got removed; was it un-needed or should we add an analogue to it?

Reply:
Good call, I'll add it back in, think I missed this in a rebase.